server/src/main/java/io/spine/server/bus/Bus.java

```diff
@@ -74,7 +76,17 @@ public void unregister(D dispatcher) {
      * @param message the message to post
      * @param responseObserver the observer to receive outcome of the operation
      */
-    public abstract void post(T message, StreamObserver<Response> responseObserver);
+    public void post(T message, StreamObserver<Response> responseObserver) {
```

Comment:
Now it looks `final`. If it cannot be declared as such, let's have a comment explaining why.